app/models/character_type.rb

```diff
@@ -1,7 +1,9 @@
 class CharacterType < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, optional: true, inverse_of: :character_types
   has_one_attached :avatar
-  has_many :character_attributes, dependent: :destroy
+  has_many :character_attributes, dependent: :destroy, inverse_of: :character_type, :autosave => true
```

Comment:
@StefanGajic use the same hash syntax, use new hash syntax, i.e. `autosave: true`.